packages/typescript/internal/ts_project.bzl

```diff
@@ -568,6 +576,18 @@ def ts_project_macro(
             will be called with the remaining arguments.
             See the packages/typescript/test/ts_project/swc directory for an example.
 
+            When a custom transpiler is used (i.e. `transpiler != "tsc"`) then this macro expands to these targets:
```

Comment:
Would a better default for `transpiler` be `None` to avoid a the magic string `tsc`? `None` implying that the default transpiler is _not_ overridden?

Reply:
sure, that's fine with me. Agree it's strange for it to accept various types, and we never intend any other string to be legal here

Reply:
Accepting different types can be useful in macros. I've used it here and there to make a public API flexible and cleaner than having different attributes.

Reply:
changing to `None` as default